src/ngx_pagespeed.cc

```diff
@@ -1265,6 +1265,22 @@ ps_loc_conf_t* ps_get_loc_config(ngx_http_request_t* r) {
       ngx_http_get_module_loc_conf(r, ngx_pagespeed));
 }
 
+RewriteOptions* ps_determine_remote_options(ps_srv_conf_t* cfg_s) {
+  if (!cfg_s || !cfg_s->server_context ||
+      !cfg_s->server_context->global_options()) {
+    return NULL;
+  }
+  if (!cfg_s->server_context->global_options()
+          ->remote_configuration_url()
+          .empty()) {
+    RewriteOptions* remote_options =
+        cfg_s->server_context->global_options()->Clone();
+    cfg_s->server_context->GetRemoteOptions(remote_options);
```

Comment:
Can you comment that this is a blocking fetch?  Things like that are very unusual here.